components/Card.js

```diff
@@ -43,41 +71,57 @@ const Card = ({
     );
     return (
         <CardView>
-            <BigScrollStyledView
-                directionalLockEnabled={true}
-                contentContainerStyle={{ maxWidth: '99.9%' }}
-            >
-                <StandardTextbox text={category.toLowerCase()} color={color} />
-
-                <CardTextbox textList={textList} color={color} />
-                {hasFollowUp ? (
-                    <TwoButtonContainer>
+            <View style={{ height: '100%' }}>
+                <View style={{ height: '9%', justifyContent: 'stretch' }}>
+                    <CardTitle text={category.toUpperCase()} color={color} />
+                </View>
+                <View style={{ height: '1.5%' }}></View>
+                <View style={{ height: '77.5%' }}>
+                    <CardTextbox textList={textList} color={color} />
+                </View>
+                <View style={{ height: '1.5%' }}></View>
+                <View style={{ height: '9%' }}>
+                    {hasFollowUp ? (
+                        <TwoButtonContainer>
+                            <CustomButton
+                                text="DONE"
+                                color={color}
+                                onPress={() => setModalVisible(false)}
+                                horizontalMargin={SMALL_HORIZONTAL_MARGIN_BUTTON}
+                                height='100%'
+                                marginVertical='0'
+                                fontWeight="500"
+                                fontSize={28}
+                            ></CustomButton>
+                            <CustomButton
+                                text="FOLLOW UP"
+                                color={color}
+                                onPress={() => setOptions(followUp)}
+                                horizontalMargin={SMALL_HORIZONTAL_MARGIN_BUTTON}
+                                height='100%'
+                                marginVertical='0'
+                                fontWeight="500"
+                                fontSize={28}
+                            ></CustomButton>
+                        </TwoButtonContainer>
+                    ) : (
                         <CustomButton
-                            text="done"
+                            text="DONE"
                             color={color}
                             onPress={() => setModalVisible(false)}
-                            horizontalMargin={SMALL_HORIZONTAL_MARGIN_BUTTON}
-                            minHeight={BIG_MIN_HEIGHT_BUTTON}
-                        ></CustomButton>
-                        <CustomButton
-                            text="follow up"
-                            color={color}
-                            onPress={() => setOptions(followUp)}
-                            horizontalMargin={SMALL_HORIZONTAL_MARGIN_BUTTON}
-                            minHeight={BIG_MIN_HEIGHT_BUTTON}
+                            horizontalMargin='20px'
```

Comment:
Should add `20px` to the `Constant.js` file and import it here in case we want to use this same spacing somewhere else

Reply:
b80a090